### PR TITLE
feat: connext v2 (testnet and mainnet)

### PR DESCRIPTION
**Pull request by reliveyy**

### CODE REVIEW

File: images/utils/launcher/node/connext.py [hunk 1]
```diff
@@ -22,13 +22,21 @@ class Connext(Node):
...
             ]
-        else:
-            environment = []
+        elif self.network == "testnet":
+            environment = [
+                "CONNEXT_NODE_URL=https://connext.testnet.odex.dev/api",
+            ]
+        elif self.network == "mainnet":
+            environment = [
+                "CONNEXT_NODE_URL=https://connext.mainnet.odex.dev/api",
```
Comment (erkarl) [inline images/utils/launcher/node/connext.py:38]: This should be `connext.odex.dev`

Comment (kilrau) [reply images/utils/launcher/node/connext.py:38]: Good catch @erkarl ! Let's use copy/paste in future.. After fixing this up I think you can take this PR out of draft for review @reliveyy

**Comment by raladev** [comment]
https://github.com/ExchangeUnion/xud-docker/blob/634ae10c8069d1557eba2cb4db9d3e6b742bf396/images/utils/launcher/node/connext.py#L50
rankibly should be used instead of ropsten

**Comment by raladev** [comment]
- [ ] when i dont pass mode for geth in testnet.conf or pass mode='native', connext does not use my native geth node. Instead of this is used https://rinkeby.indra.connext.network/api/ethprovider.

Expected result:
- connext client used my local geth node;
- till geth sync, status of connext should be 'Wait geth sync'.
[Screenshot from 2020-05-28 17-39-59](https://user-images.githubusercontent.com/29906866/83157260-0f80cc00-a10c-11ea-84fe-1a9dc240709f.png)
[Screenshot from 2020-05-28 17-39-08](https://user-images.githubusercontent.com/29906866/83157287-17407080-a10c-11ea-855f-a362c21e0195.png)

**Comment by kilrau** [comment]
Could you resolve these conflicts? @reliveyy

**Comment by reliveyy** [comment]
> Could you resolve these conflicts?

Done.

**Pull request merged**